dev/com.ibm.websphere.appserver.cdi2.0-appSecurity1.0/com.ibm.websphere.appserver.cdi2.0-appSecurity1.0.feature

```diff
@@ -2,7 +2,7 @@
 symbolicName=com.ibm.websphere.appserver.cdi2.0-appSecurity1.0
 visibility=private
 IBM-Provision-Capability: osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=com.ibm.websphere.appserver.cdi-2.0))", \
- osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=com.ibm.websphere.appserver.appSecurity-2.0))"
+ osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=com.ibm.websphere.appserver.appSecurity-3.0))"
```

Comment:
Can this auto-feature be renamed?

Reply:
I was planning to do a second pass for cosmetic renames after this PR.  Since there are quite a lot of thing changing in this PR, I didn't want to add a bunch of renames on top of that.